syntaxes/asterisk.tmLanguage

```diff
@@ -83,6 +83,35 @@
 				</dict>
 			</dict>
 
+			<!-- 'same' keywords continues an extension declaration with priorities and priority labels-->
+			<dict>
+				<key>match</key>
+				<string>(^same  =>?)(\s+(n|\d+)\b(\(([\w\d_-]+)\))?)?</string>
+				<key>captures</key>
+				<dict>
+					<!-- Continues an extension declaration -->
+					<key>1</key>
+					<dict>
+						<key>name</key>
+						<string>keyword.other</string>
+					</dict>
+
+					<!-- Priority -->
+					<key>2</key>
+					<dict>
+						<key>name</key>
+						<string>constant.numeric</string>
+					</dict>
+
+					<!-- Priority label -->
+					<key>5</key>
+					<dict>
+						<key>name</key>
+						<string>constant.character</string>
```

Comment:
In the exten match I set the priority label sub-match to "constant.numeric" to maintain the same highlighting colour as the priority number. Propose changing this to "constant.numeric" OR changing the exten priority label to "constant.character" for consistency.

Reply:
You're right. I forgot this one.
In fact I am preparing another PR in which I modified some of the types (in this one I propose to change the label from numeric to character).
I was going to do it in another PR because it was only cosmetic changes.

So in this PR I'll get back to `constant.numeric` for the label of the `same` match.
I'll send you the other PR soon (BTW I would also like to send you a 3rd PR in which I update the Application and Function dict).

Reply:
OK that sounds good, and I look forward to seeing your next PR. Some of the scopes I used were not semantically correct, but none of the included highlighting themes had enough range to show the contrast I wanted. In hindsight I think it's probably best to make the scopes correct and leave it for themes to highlight what they will.